src/adhocracy_frontend/adhocracy_frontend/static/js/Packages/ResourceActions/ResourceActions.html

```diff
@@ -34,4 +36,24 @@
         data-class="action-bar-item"
         data-parent-path="parentPath"
         data-resource-path="{{resourcePath}}"></adh-cancel-action>
+</div>
+
+<ul class="moving-column-alerts">
+    <li data-ng-repeat="(id, alert) in modals.alerts" class="moving-column-alert m-{{alert.mode}}" data-ng-click="modals.removeAlert(id)">
+        {{ alert.message | translate }}
+    </li>
+</ul>
+
+<div class="action-bar-modal" data-ng-if="modals.overlay">
+    <adh-report-abuse
+        data-ng-if="modals.overlay === 'abuse'"
+        data-modals="modals"
+        class="report-abuse"
+        data-url="{{ proposalUrl | adhResourceUrl | adhCanonicalUrl }}">
```

Comment:
Why did you leave `proposalUrl` and didn't change it to `resourcePath`? (It seems to work fine, but I thought `resourceActions` are not just about proposals.)